### DISCUSSION TITLE
Export du nombre de places restantes

### DISCUSSION THREAD

**Discussion by LHuberson** [unistra/esup-immersup-demandes#18] [6- Administration]
Est il possible de voir en direct le nombre global de places restantes? (au global / par composante/ par département)

**Comment by efinkbeiner5** [comment]
Dans quel menu tu voyais cela ? Il faudrait trier par période je suppose puis par structure ?  faut-il différencier cours et évènements ? 
Tu veux juste le nombre global de places en tout pour cours et évènements ?

**Comment by LHuberson** [comment]
Aujourd'hui, je faisais une extraction > Données anonymisées des cours > Calcul nombre de place -  nombre d'inscription 
S'il y avait un compteur des places restantes (en prenant en compte uniquement les cours à venir) éventuellement par domaine de formation voir sous domaine ça serait top. 
Pour les évènements nous n'avons pas le même besoin pour le moment.

**Reply by efinkbeiner5** [reply]
Si on fait une nouvelle page dans les statistiques avec des filtres périodes, domaines, sous domaines, structures et qu'on affiche le nombre de places restantes à venir ou globalement, ça pourrait convenir ?

**Comment by LHuberson** [comment]
Oui je pense. Merci

**Comment by efinkbeiner5** [comment]
Nouvelle page dans les statistiques avec un filtre par période, un filtre domaine, sous domaines, établissement, structures (pour les cours)
Pour les évènements, un filtre par période, type d'évènement, établissement, structures. 

1- Ajouter une table des groupements d'établissement
2- Modifier les fiches lycées pour permettre de lier les lycées à un ou plusieurs groupements d'établissement
3- Modifier le formulaire de création de créneaux pour pouvoir utiliser les groupements d'établissement comme nouveau type de restriction pour gérer les restrictions par groupements
4 - modification de la partie publique pour gérer le nouveau type de restriction (cours, évènement et moteur), affichage de la restriction, affichage du bouton s'inscrire. 

Complexité : modérée
Chiffrage : 7 jours

**Comment by efinkbeiner5** [comment]
Non réalisé en 2025
